Modules/IO/ImageBase/include/itkIOCommon.h

```diff
@@ -34,7 +34,6 @@ class IOCommonEnums
 {
 public:
   /**
-   * \ingroup IOFilters
```

Comment:
I guess that this arose from refactoring into strongly typed enums?

Reply:
https://github.com/InsightSoftwareConsortium/ITK/commit/0dd31a62f851a0ae95ef4c6d6c85e531a4ebca59#diff-0d8935a3dec9fa237c6147802f497a673e0010617f43c990bc727fb7181aca95R37